crowbar_framework/app/models/service_object.rb

```diff
@@ -776,9 +779,10 @@ def proposal_commit(inst, in_queue = false, validate_after_save = true)
         response = [500, e.message]
       ensure
         # Make sure we unmark the wall
-        prop = ProposalObject.find_proposal(@bc_name, inst)
+        prop = Proposal.where(barclamp: @bc_name, name: inst).first.reload
```

Comment:
> Why `reload`?

The repeated find on ProposalObject fetched all the changes made since
the first call at the top of the method.

So reload does the same - otherwise, rails would see that it already
fetched the model and simply re-save it, IIUC.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/crowbar/barclamp-crowbar/pull/1269/files#r29846247